java/util/HashMap/HashMap.hpp

```diff
@@ -442,17 +491,22 @@ namespace Java {
 							String colonAndSpace = ": ";
 							String endString = "}";
 							String totalString;
+							std::string str;

```

Comment:
I can't find the usage of this variable. Please check it. Remove if you don't need it. Thanks.